docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc

```diff
@@ -85,13 +87,13 @@ docker run \
   --rm docker.elastic.co/beats/elastic-agent:{version}
 ----
 <1> Your cluster's {es} host URL
-<2> The Fleet service token -- generate one in the Fleet UI if you don't have one already
-<3> ID of the Fleet Server policy. To learn how to create a policy, refer
+<2> The {fleet} service token -- generate one in the {fleet} UI if you don't have one already
+<3> ID of the {fleet-server} policy. To learn how to create a policy, refer
 <<create-a-policy-no-ui>>.
 
 We recommend only having one fleet-server policy.
 
-NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. Refer to {observability-guide}/uptime-set-up.html[Synthetics Fleet Quickstart] for more information.
+NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. Refer to {observability-guide}/uptime-set-up.html[Synthetics {fleet} Quickstart] for more information.
```

Comment:
Not sure if we want to use an attribute in the titles of other guides.

```suggestion
NOTE: Replace `docker.elastic.co/beats/elastic-agent` with `docker.elastic.co/beats/elastic-agent-complete` and use the `elastic-agent` user instead of root to run Synthetics Browser tests. Synthetics tests cannot run under the root user. Refer to {observability-guide}/uptime-set-up.html[Synthetics Fleet Quickstart] for more information.
```